cookbooks/fb_networkmanager/attributes/default.rb

```diff
@@ -0,0 +1,43 @@
+#
+# Cookbook:: fb_networkmanager
+# Recipe:: default
+#
+# Copyright (c) 2019-present, Vicarious, Inc.
+# Copyright (c) 2019-present, Facebook, Inc.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+default['fb_networkmanager'] = {
+  'enable' => false,
+  'system_connections' => {},
+  'config' => {
+    'main' => {
+      'plugins' => [
+        'ifupdown',
+        'keyfile',
+      ],
+    },
+    'ifupdown' => {
+      # yup... this boolean does NOT take true/false like others,
+      # but instead yes/no.
+      #
+      # NetworkManager is the worst.
+      'managed' => 'no',
+    },
```

Comment:
Agree that this is clowny, but why expose it in the API? In other cookbooks we just expose a boolean here and translate it to the appropriate string in the template, I think we should do the same here.

Reply:
Because SOME things are booleans and want `true`/`false` and SOME things want `yes`/`no`. My original version did just as you describe and it broke. That's the point of the comment.

Rather than try to hardcode the entirety of every option and what it wants, which is the antithesis of how FB-API cookbooks work, I'm leaving it up to the user to pass in the right one.

Reply:
ffs, ok, they I'd recommend expanding that comment and adding a note in the README